src/core/util/util.spec.js

```diff
@@ -443,6 +443,22 @@ describe('util', function() {
       flush(function(){ expect( callBackUsed ).toBeUndefined(); });
     }));
 
+    it('should only skip callbacks of scopes which were destroyed', inject(function($mdUtil) {
+      var callBackUsed1, callback1 = function(){ callBackUsed1 = true};
+      var callBackUsed2, callback2 = function(){ callBackUsed2 = true};
+      var scope1 = $rootScope.$new(true);
+      var scope2 = $rootScope.$new(true);
+
+      $mdUtil.nextTick(callback1, false, scope1);
```

Comment:
Yea, if it failed previously and it's a case which is not covered yet, then it would make definitely sense.